### DISCUSSION TITLE
防火墙没有自定义规则，是需要安装某个包才能修改吗？

### DISCUSSION THREAD

**Discussion by 14790897** [immortalwrt/immortalwrt#1515] [Q&A]
![image](https://github.com/user-attachments/assets/eda5895e-1676-48ec-8171-1ec73a23b18d)

**Comment by 14790897** [comment]
求助

**Comment by 14790897** [comment]
继续求助

**Reply by cander0815** [reply]
我个人目前用的luck做端口转发，它好像会默认修改防火墙规则。支持ipv6的转发到ipv4的端口

我参考的这个文章，https://post.smzdm.com/p/agqg8w7w/#cl_4，你看复不符合你的需求。

**Reply by 14790897** [reply]
谢谢，但我是要设置iptables -t nat -I POSTROUTING -j MASQUERADE

**Reply by qiqiandfei** [reply]
> 谢谢，但我是要设置iptables -t nat -I POSTROUTING -j MASQUERADE

解决了吗？遇到了同样的问题~

**Reply by 14790897** [reply]
没有

**Reply by qiqiandfei** [reply]
> 没有

试试这个配置
![image](https://github.com/user-attachments/assets/59a72e90-33be-434c-b09e-2be91125f812)

**Comment by Fvpder** [comment]
23.05开始使用firewall4使用nftables防火墙，开启防火墙里的ip动态伪装就好了
